README.md

```diff
@@ -28,38 +28,38 @@ How to build and run Docker images
 Each combination of ServiceControl instance, transport, and topology has a dedicated dockerfile. Select the instance, transport, and topology you want to run and build the `init` container and the runtime container by executing the following commands (using RabbitMQ Conventional topology as an example) from within the `src\docker` folder:
 
 ```
-docker build -f .\dockerfile.rabbitmq.conventional.init -t particular/servicecontrolrabbitconventional.init ./../
-docker build -f .\dockerfile.rabbitmq.conventional -t particular/servicecontrolrabbitconventional ./../
-docker build -f .\dockerfile.rabbitmq.conventional.audit.init -t particular/servicecontrolrabbitconventional.audit.init ./../
-docker build -f .\dockerfile.rabbitmq.conventional.audit -t particular/servicecontrolrabbitconventional.audit ./../
-docker build -f .\dockerfile.rabbitmq.conventional.monitoring.init -t particular/servicecontrolrabbitconventional.monitoring.init ./../
-docker build -f .\dockerfile.rabbitmq.conventional.monitoring -t particular/servicecontrolrabbitconventional.monitoring ./../
+docker build -f .\dockerfile.rabbitmq.conventional.main.init        -t particular/servicecontrolrabbitconventional.main.init ./../
```

Comment:
@WilliamBZA maybe main should be error?

Reply:
I used these UI elements to guide me, but the naming of these things has always been a bit awkward:

<img src="https://user-images.githubusercontent.com/677704/99953515-a0fe8d00-2d81-11eb-9b4b-f8cca635af15.png" width="702px" />

<img src="https://user-images.githubusercontent.com/677704/99953552-b1af0300-2d81-11eb-84e4-560d0b673231.png" width="203px" />